### ISSUE TITLE
Accessibility: "Aria-labelledby" has incorrect (ID) reference - Slideshow

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#415]
**Description**
1. [Apples to Cider](https://exhibits.library.cornell.edu/cider/feature/old-fruit-new-light) has this issue on the slideshow gallery.

**To Reproduce**
1. Go to the page and look at the slideshow code. There is no label in the modal that is associated with this.

**How to fix**
Option 1: Adding a title does not fix this issue, as it doesn't seem to add it to the modal window. Would recommend adding a field to be able to add a slideshow title and then create the label within the slideshow modal code.
Option 2: Remove the slideshow option. According to the [WCAG Modal Dialog Example](https://www.w3.org/TR/wai-aria-practices-1.1/examples/dialog-modal/dialog.html), all modals need to have some form of label.

**Additional Issue**
1. The slideshow that is being used auto-plays when a user opens it. We cannot have autoplaying slideshows without a sufficient amount of time for each slide. @mhk33 might know more about this since the current Olin/Uris site uses a rotating gallery right now.

**How to fix - Additional Issue**
Option 1: Do not allow users to use the slideshow option at all. We made the decision to not allow this on CUL unit websites that have been migrated to WordPress because this can be a more subjective and difficult issues to fix
Option 2: Allow enough time with an auto-rotating slideshow so that each slide would be up for a significant amount of time. This would allow users with screen readers and keyboard navigation to be able to access this information.

**Comment by mhk33** [comment]
I think on Olin-Uris we removed the auto-slideshow capability and made sure to have controls so the user could navigate through the slideshow themselves. The controls would need to be keyboard-accessible.
